apidoc/Titanium/Network/HTTPClient.yml

```diff
@@ -536,6 +564,34 @@ properties:
         On iOS, the default is 15000 (15 seconds).
     type: Number
 
+  - name: timeoutForResource
+    summary: |
+        The maximum amount of time (in milliseconds) that a resource request should be allowed to take.
+    description: |
+        The resource timeout controls how long (in milliseconds) to wait for an entire resource to transfer before giving up.
+        The resource timer starts when the request is initiated and counts until either the request completes or this timeout is reached, whichever comes first.
+        For more detail see [timeoutintervalforresource](https://developer.apple.com/documentation/foundation/nsurlsessionconfiguration/1408153-timeoutintervalforresource?language=objc)
+        from Apple documentation.
+        This property also applies if <Ti.Network.HttpClient.waitsForConnectivity> is set to `true`.
```

Comment:
I have to say that this property is still fairly confusing to me. I suppose the unit test example helps explain it a little better to me - so maybe we should add an example in the docs?

It looks like a request would timeout if the minimum value of this property *OR* the `timeout` property were reached? (i.e. if `timeout` is 10 seconds and this is 5 seconds, we'd time out after 5 seconds. If this is 10 seconds and `timeout` is 7 seconds, we'd time out after 7 seconds?)

And the big difference here is when used in conjunction with `waitsForConnectivity` where `timeout` would "start" once connectivity/request begins, but `timeoutForResource` would "start" counting once you scheduled the request (i.e. if we lose connectivity and fire off a request, this timer starts immediately while `timeout` wouldn't start until we regained connectivity and actually sent off the request?)

Again, all of this is to say it may be useful to try and do some more explaining in the docs here to help people understand the difference between the properties and how they interact - because I'm just guessing at the above.